src/Substrait.Core/Substrait.Core.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <ProtoBufNet_Debug>True</ProtoBufNet_Debug>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="../../substrait/proto/substrait/**/*.proto" ProtoRoot="../../substrait/proto/" CompileOutputs="false" OutputDir="generated/proto/" GrpcServices="None" />
```

Comment:
Why `CompileOutput="false"` instead of letting it compile and generate straight to the build directory?  I can see some potential advantages (e.g. being able to explore the generated source more easily) but was curious if you had more reasoning behind this?